### PR TITLE
feat: example integration with React app

### PR DESCRIPTION
**Pull request by josadcha**
Adds `Example Integration with React` section to the SDK docs with a "drop-in code" with txes:
- approve
- deposit
- requestRedeem

This is how it would look in a browser:
Screenshot 2024-11-01 at 17 23 00

### CODE REVIEW

File: technical-resources/sdk/usage-guide.md [hunk 1]
```diff
@@ -75,3 +75,170 @@ import { pool } from '@maplelabs/maple-js';
...
+
+   Below is a complete example of a React component that interacts with the Maple Protocol:
+
+   ```typescript
+   import React, { useEffect, useState } from 'react';
+   import { BigNumber, ethers } from 'ethers';
+   import { addresses, poolV2, environmentMocks } from '@maplelabs/maple-js';
+
+   const PROJECT = 'mainnet-prod';
+   const POOL_ADDRESS = '0xpoolAddress';
```
Comment (callum-hyland) [inline technical-resources/sdk/usage-guide.md:112]: Should we provide a valid pool ID which has the last pool requirements?

**Comment by callum-hyland** [comment]
We'll also need to mention the onchain flag requirement somewhere, as that's not available from the SDK (yet)

File: technical-resources/sdk/usage-guide.md (hunk 1 above)
Comment (josadcha) [reply technical-resources/sdk/usage-guide.md]: updated with Pool address and new section `Checking Permissions and Onboarding`

**Review by callum-hyland — APPROVED** [review]

**Review by michaelsmueller — APPROVED** [review]
🤌

**Pull request merged**